components/sensor/kalman_combinator.rst

```diff
@@ -0,0 +1,61 @@
+Kalman filter-based sensor fusion
+=================================
+
+.. seo::
+    :description: Instructions for setting up a kalman_combinator sensor
+
+The ``kalman_combinator`` sensor platform allows you to filter one or several
+sensors into one with a reduced error. If using a single sensor as data source,
+it acts like a :ref:`sensor-filter-exponential_moving_average` filter. With
+multiple sensors, it combines their values based on their respective standard
+deviation.
+
+The ``unit_of_measurement``, ``device_class``, ``entity_category``, ``icon``, and
+``accuracy_decimals`` properties are by default inherited from the first sensor.
+``state_class`` is explicitly not inherited, because ``total_increasing`` states
+could still decrease when multiple sensors are used.
+
+.. code-block:: yaml
+
+    # Example configuration entry
+    sensor:
+      - platform: kalman_combinator
+        name: "Temperature"
+        unit_of_measurement: °C
+        process_std_dev: 0.001
+        sources:
+          - source: temperature_sensor_1
+            error: 1.
```

Comment:
The trailing decimal point is a little confusing
```suggestion
            error: 1
```

Reply:
Maybe 1.0 instead? I just want to show that it's a float.

Reply:
Yeah, let's do that.
```suggestion
            error: 1.0
```